controllers/userControllers.js

```diff
@@ -0,0 +1,44 @@
+import { Todos } from '../models/todo.js';
+
+
+const insertActionTodo = async (req, res) => {
+    const { username, group, subtask, isUrgent, isDone, hours } = req.body;
+
+    try {
+        const todoList = await Todos.create({
+            group,
+            subtask,
+            isUrgent,
+            isDone,
+            hours
+        });
+        return res.status(201).json({ message: `Information is inserted to database by user ${username}`, task: todoList });
+    } catch (err) {
+        console.log("Error inserting todoList!");
+        res.status(500).json({ message: "Failes to insert todoList!" });
+    }
+
+};
+
+const updateTask = async (req, res) => {
+    const { task_id, isDone } = req.body;
+    try {
+        if (task_id == null) {
+            res.status(400).json({ message: "Task id is required, please provide a valid task id." });
+            return;
+        }
+        const updated = await Todos.update({ isDone: isDone }, {
```

Comment:
You can just write {isDone}, and {task_id}. The variable and the parameter names are the same here.

Reply:
I dont understand, we can only change isDone column. Why we should update also task_id? Am I right?